src/components/Post.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { User } from './User';
+import { CommentList } from './CommentList';
+import { PostType } from '../types';
+
+export const Post = ({ id, body, title, author, comments }) => (
+  <div className="App__post-body">
+    <div className="App__post-header">
+      <h2 className="App__title">
+        Post
+        {' '}
+        {id}
+        :
+        {' '}
+        <i>{title}</i>
+      </h2>
+      <article className="App__article">{body}</article>
+      <div>
+        <User {...author} />
+      </div>
+    </div>
+
+    <ul>
+      <h4>Comments:</h4>
+      <CommentList comments={comments} />
+    </ul>
+  </div>
+);
+
+Post.propTypes = PostType;
```

Comment:
```suggestion
Post.propTypes = {
  userId: PropTypes.number.isRequired,
  id: PropTypes.number.isRequired,
  title: PropTypes.string.isRequired,
  body: PropTypes.string.isRequired,
};
```

Reply:
If you receive a destructured post, you receive not a shape of post, but separate properties, and need to describe type of each of them separately

Reply:
fixed